### PR TITLE
Added UI Tests

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Added UI Tests
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| Sponsor company   | @PrestaShopCorp
| How to test?      | CI is :green_circle:

### CODE REVIEW

**Review by nesrineabdmouleh — APPROVED** [review]

File: .github/workflows/ui-test.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  ## Check each PR
+  push:
+  pull_request:
+  ## Manual execution on branch
+  workflow_dispatch:
+  ## Nightly
+  ### Needs secrets
+  #### GC_PROJECT_ID
+  #### GC_SERVICE_KEY
+  #### NIGHTLY_TOKEN
```
Comment (boubkerbribri) [inline .github/workflows/ui-test.yml:12]:
```suggestion
```

File: .github/workflows/ui-test.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        working-directory: tests/UI/
+        if: ${{ github.event_name == 'schedule' }}
+        run: gsutil cp -r "nightly/**" gs://prestashop-core-nightly/reports
+
+      # Nightly : Push Report
+      - name: "Nightly : Push Report"
+        if: ${{ github.event_name == 'schedule' }}
+        run: |
+          REPORT_NAME="blockwishlist_$(date +%Y-%m-%d)-${{ env.PS_VERSION }}"
+          curl -v "https://api-nightly.prestashop-project.org/import/report/playwright?token=${{ secrets.NIGHTLY_TOKEN }}&filename=${REPORT_NAME}.json"
```
Comment (boubkerbribri) [inline .github/workflows/ui-test.yml:106]:
```suggestion
          curl -v "https://api-nightly.prestashop-project.org/import/report/playwright?token=${{ secrets.NIGHTLY_TOKEN }}&filename=${REPORT_NAME}.json"

```

File: tests/UI/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      "paths": {},
+      "typeRoots": [
+        "node_modules/@types"
+      ],
+      "skipLibCheck": true,
+      "noEmit": true
+    },
+    "include": ["types/global.d.ts", "campaigns/**/*"],
+    "exclude": ["node_modules"]
+  }
```
Comment (boubkerbribri) [inline tests/UI/tsconfig.json:24]:
```suggestion
  }

```

File: tests/UI/types/global.d.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  var PSConfig: GlobalPSConfig;
+  var BROWSER: GlobalBrowserConfig;
+  var GENERATE_FAILED_STEPS: any;
+  var SCREENSHOT: GlobalScreenshot;
+  var maildevConfig: GlobalMaildevConfig;
+  var keycloakConfig: GlobalKeycloakConfig;
+  var browserErrors: GlobalBrowserErrors;
+}
+
+export {};
```
Comment (boubkerbribri) [inline tests/UI/types/global.d.ts:27]:
```suggestion
export {};

```

File: .github/dependabot.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+      day: monday
+      time: "04:00"
+    open-pull-requests-limit: 10
+    target-branch: dev
+    reviewers:
+      - "PrestaShop/qa-automation"
+    labels:
+      - "dependencies"
+      - "E2E Tests"
+      - "TE"
```
[the remark this answers is not in the document]
Comment (Progi1984) [reply .github/dependabot.yml:16]: Fixed

File: .github/workflows/ui-test.yml (hunk 1 above)
Comment (Progi1984) [reply .github/workflows/ui-test.yml:12]: No, it's wanted for documentation.

File: .github/workflows/ui-test.yml (hunk 2 above)
Comment (Progi1984) [reply .github/workflows/ui-test.yml:106]: Fixed

File: tests/UI/tsconfig.json (hunk 3 above)
Comment (Progi1984) [reply tests/UI/tsconfig.json:24]: Fixed

File: tests/UI/types/global.d.ts (hunk 4 above)
Comment (Progi1984) [reply tests/UI/types/global.d.ts:27]: Fixed

**Pull request merged**